### PR TITLE
install kubectl on github action machines

### PR DESCRIPTION
**Pull request by medyagh**
this will install kubectl on github action CI machines as 
some of our tests depends on having kubectl.

aslo adds more docker info for debuggings

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -157,6 +170,11 @@ jobs:
       SHELL: "/bin/bash" # To prevent https://github.com/kubernetes/minikube/issues/6643
     needs: [build_minikube]
     steps:
+    - name: Install kubectl
+      shell: bash
+      run: |
+        curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl
+        sudo install kubectl /usr/local/bin/kubectl
```
Comment (afbjorklund) [inline .github/workflows/main.yml]: do you need to make it executable too ?

https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -232,6 +258,11 @@ jobs:
       SHELL: "/bin/bash" # To prevent https://github.com/kubernetes/minikube/issues/6643
     runs-on: ubuntu-16.04
     steps:
+    - name: Install kubectl
+      shell: bash
+      run: |
+        curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl
```
Comment (afbjorklund) [inline .github/workflows/main.yml]: chmod +x kubectl

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -376,11 +412,11 @@ jobs:
         SHELL: "/bin/bash" # To prevent https://github.com/kubernetes/minikube/issues/6643
       runs-on: ubuntu-18.04
       steps:
-      - name: Install lz4
+      - name: Install kubectl
         shell: bash
         run: |
-          sudo apt-get update -qq
-          sudo apt-get -qq -y install liblz4-tool
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl
```
Comment (afbjorklund) [inline .github/workflows/main.yml]: chmod +x kubectl

File: .github/workflows/main.yml (hunk 1 above)
Comment (medyagh) [reply .github/workflows/main.yml]: I think install makes it executable. doesnt it ?

Comment (afbjorklund) [reply .github/workflows/main.yml]: Ah, right. They used `mv` or `cp`. Maybe install fixes the owner, too

**Pull request merged**